demos/projects/ST/b-l475e-iot01a/config/demo_config.h

```diff
@@ -13,7 +13,7 @@
  * This Model ID is tightly tied to the code implementation in `sample_azure_iot_pnp_simulated_device.c`
  * If you intend to test a different Model ID, please provide the implementation of the model on your application.
  */
-#define sampleazureiotMODEL_ID                                "dtmi:com:example:Thermostat;1"
+#define sampleazureiotMODEL_ID                                "dtmi:azure:iot:deviceUpdateModel;1"
```

Comment:
this will break the other PnP samples. I have here in the Linux PR a way to fix it but unfortunately that one will take a little more time to get through:

https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/01e8c3a4534560971bd68a7fd8f6c8fb8f6b8e17/demos/sample_azure_iot_adu/sample_azure_iot_adu.c#L204-L207

Reply:
I think this is fine for now but can you add a TODO to change this once the Linux PR is in?